app/views/shared/_spam_honeypot.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# Honeypot field intended to reduce spam %>
+<div class="visually-hidden" aria-hidden="true">
+  <label for="val">Ignore if you're not a robot</label>
+  <input type="text" tabindex="-1" autocomplete="off" id="val" name="<%= "#{form_name}" %>[val]"/>
```

Comment:
I think this should be fine as it is. In case screen readers still see the content (which I doubt but haven't tested), the text makes it clear to ignore.
If you want us (the accessibility team) to test that with various screen readers, we could do it, but that would need to be prioritised. In any case, it looks like at best everything will be fine, at worst it will be a minor annoyance, but definitely not a barrier.